src/codegen/sys/cargo_toml.rs

```diff
@@ -39,7 +39,9 @@ fn fill_empty(root: &mut Table, env: &Env, crate_name: &str) {
         set_string(
             package,
             "links",
-            nameutil::crate_name(&env.config.library_name),
+            // FIXME: Will pick the first library from list.
+            // links = ["foo", "bar"] doesn't work...
```

Comment:
Remove the `FIXME` comment from here, that workaround is in the function itself and not here :)